app/modules/wifi.c

```diff
@@ -730,45 +788,10 @@ static int wifi_station_setauto( lua_State* L )
   a = luaL_checkinteger( L, 1 );
   luaL_argcheck(L, ( a == 0 || a == 1 ), 1, "0 or 1");
   wifi_station_set_auto_connect(a);
-  return 0;  
+  return 0;
```

Comment:
Oh, looks like your editor fixes whitespace automatically just like mine 😉 If I didn't have so little extra time atm I'd finally tackle https://github.com/nodemcu/nodemcu-firmware/issues/1168#issuecomment-201486819 to fix this.